jbang/aws-bedrock-ingestion/README.adoc

```diff
@@ -166,6 +166,50 @@ curl http://0.0.0.0:8080/bedrock/prompt/
 The average price of natural gas between 2010 and 2011 was $3.93 per unit.
 ----
 
+=== Running a full session with the Knowledge Base
+
+It's interesting to use the ability to create a session with the Knowledge Base.
+
+To do this there is a route you can run, composed of Kamelets, named `aws-bedrock-knowledgebase-session.camel.yaml`
+
+This route will will be triggered by a timer and then it will loop, asking random range average value for natural gas prices, re-using the same Session id.
```

Comment:
```suggestion
This route will be triggered by a timer and then it will loop, asking random range average value for natural gas prices, re-using the same Session id.
```

Reply:
I committed the fix separately.

Reply:
Thanks @apupier